Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/ToolbarItems/Common/CommonButtons.Events.cs

```diff
@@ -109,6 +109,28 @@ private void MakeOList(ToolbarButton button)
 
                 string linkText = linkBox.Text.Trim();
 
+                if (string.IsNullOrEmpty(labelText.Trim()))
```

Comment:
This could probably be one ContentDialog, then change the Content accordingly.

Reply:
@WilliamABradley, updated the code to show common ContentDialog with different warning message.